docs/user/man_pages/bodhi.rst

```diff
@@ -271,6 +271,11 @@ The ``updates`` command allows users to interact with bodhi updates.
 
         The name of the update
 
+    ``--from-tag``
+
+        If this flag is provided ``<builds>`` are handled as Koji tag. We support only single Koji tag
+        at once. Koji tag allows to bundle multiple builds together in one update.
+
```

Comment:
Similar here (`builds` -> e.g. `builds_or_tag`, whatever click makes out of the chosen variable name). Also, I'd reword it like this:

```
If this flag is provided, `builds_or_tag` will be interpreted as a Koji tag and expand to all latest builds in it. Only a single tag can be provided.
```

Also, I don't see that we ensure only one tag is specified by the user, would you add this (see above)?